src/DynamoCore/Configuration/Configurations.cs

```diff
@@ -429,5 +429,12 @@ internal enum Units
         }
 
         #endregion
+
+        #region Notifications Constants
+        /// <summary>
+        /// Request timeOut for notifications service
+        /// </summary>
+        public const int NotificationsDefaultTimeOut = 10000;
```

Comment:
Ruby defaults to 60 seconds, and [.NET defaults to 100s](https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httpclient.timeout?view=net-7.0#remarks) I think the current value is a good one for time saving purpose but not overly aggresive